modules/govuk_jenkins/templates/jobs/smokey.yaml.erb

```diff
@@ -29,13 +29,16 @@
             predefined-parameters: |
                 NSCA_CHECK_DESCRIPTION=<%= @service_description %>
                 NSCA_OUTPUT=<%= @service_description %> failed
+    parameters:
+      - string:
+          name: TARGET_APPLICATION
+          description: Application to target (optional).
     builders:
       - shell: |
           set +x
           export TARGET_PLATFORM=<%= @environment %>
-          export MYTASK=<%= @smokey_task %>
-          set -x
           ./jenkins.sh
+          set -x
```

Comment:
How come `set -x` is moved after executing `./jenkins.sh`?

Reply:
Ah, that was a quick change to see if we needed it in `jenkins.sh` as well (we do, as it doesn't persist between processes). I left it here, as it was weird to only be echoing some of the commands, although I realise I didn't explain why 😬. Shall I move it back?

Reply:
No problem! Yeah I think move it back because its a change not really relevant to the scope of this PR.

Reply:
Sure, updated.